ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -78,6 +107,7 @@ public ActionResult DisplaySinglePackage(Package package)
         // GET: PackagesController/Edit/5
         public ActionResult Edit(int packageNumber)
         {
+            _memoryRepository.LoadData();
```

Comment:
To jest akurat już zmienione przeze mnie. Nie pamiętam, czy wypuściłem zmiany z tym, ale nie musisz tego używać teraz, bo dane są wczytywane na początku działania programu. Wejdź sobie na moją gąłąź i podejrzyj. Zmienimy to

Reply:
Ok